lib/SIL/CMakeLists.txt

```diff
@@ -41,9 +41,6 @@ add_swift_library(swiftSIL STATIC
 )
 
 # intrinsics_gen is the LLVM tablegen target that generates the include files
-# where intrinsics and attributes are declared. See the comment in lib/AST for
-# more detail.
-if(NOT SWIFT_BUILT_STANDALONE)
-  get_property(CLANG_TABLEGEN_TARGETS GLOBAL PROPERTY CLANG_TABLEGEN_TARGETS)
-  add_dependencies(swiftSIL intrinsics_gen ${CLANG_TABLEGEN_TARGETS})
-endif()
+# where intrinsics and attributes are declared. swiftSIL depends on these
+# headers.
+add_dependencies(swiftSIL intrinsics_gen clang-tablegen-targets)
```

Comment:
Can you put back the `See the comment in lib/AST`. Also, can you add a see the comment in lib/CMakeLists.txt?